<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>817</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>861</width>
      <height>741</height>
     </rect>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>801</width>
       <height>691</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/wallpaperMountain.jpg);
border-radius: 40px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>781</width>
       <height>651</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,100);
border-radius: 30px;</string>
     </property>
     <property name="text">
      <string comment="Password"/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>100</y>
       <width>221</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>40</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgba(90, 233, 235, 1);</string>
     </property>
     <property name="text">
      <string>Log In</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nombreUsuarioLogin">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>250</y>
       <width>531</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 0, 0, 0);
border: none;
border-bottom: 4px solid rgba(105,118,132,255);
color: rgba(255,255,255,255);
padding-bottom: 7px;</string>
     </property>
     <property name="placeholderText">
      <string>DNI del Usuario</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="contrasenaLogin">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>360</y>
       <width>531</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 0, 0, 0);
border: none;
border-bottom: 4px solid rgba(105,118,132,255);
color: rgba(255,255,255,255);
padding-bottom: 7px;</string>
     </property>
     <property name="placeholderText">
      <string>Contraseña</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loginButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>480</y>
       <width>531</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#loginButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(20, 47, 78, 219), stop:1 rgba(85, 98, 112, 226));
	color: rgba(255,255,255,210);
	border-radius: 10px;
}
QPushButton#loginButton:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(40, 67, 98, 219), stop:1 rgba(105, 118, 132, 226));
}
QPushButton#loginButton:pressed{
	padding-left: 5px;
	padding-top: 5px;
	background-color:rgba(105,118,132,200);
}</string>
     </property>
     <property name="text">
      <string>Log In</string>
     </property>
    </widget>
    <widget class="QPushButton" name="registerButtonLogin">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>550</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#registerButtonLogin{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(120, 47, 78, 219), stop:1 rgba(85, 98, 112, 226));
	color: rgba(255,255,255,210);
	border-radius: 10px;
}

QPushButton#registerButtonLogin:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(40, 167, 98, 219), stop:1 rgba(105, 118, 132, 226));
}

QPushButton#registerButtonLogin:pressed{
	padding-left: 5px;
	padding-top: 5px;
	background-color:rgba(105,118,132,200);
}</string>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
    <widget class="QPushButton" name="recuperarContrasenaButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>550</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#recuperarContrasenaButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(20, 47, 178, 219), stop:1 rgba(85, 98, 112, 226));
	color: rgba(255,255,255,210);
	border-radius: 10px;
}

QPushButton#recuperarContrasenaButton:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(40, 67, 98, 19), stop:1 rgba(105, 118, 132, 226));
}

QPushButton#recuperarContrasenaButton:pressed{
	padding-left: 5px;
	padding-top: 5px;
	background-color:rgba(105,118,132,200);
}</string>
     </property>
     <property name="text">
      <string>¿Olvidaste la Contraseña?</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../../../../../AppData/Local/Packages/PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0/LocalCache/local-packages/Python310/site-packages/qt5_applications/Qt/bin/res.qrc"/>
 </resources>
 <connections/>
</ui>
